### DISCUSSION TITLE
CSS styles to shadow DOM for the Table component

### DISCUSSION THREAD

**Discussion by manoj23sap** [UI5/webcomponents-react#5295] [Show and tell]
After Appling the CSS styles to shadow DOM for the Table component, styles are not coming as expected. 
![Screenshot 2023-11-30 at 10 54 36 AM](https://github.com/SAP/ui5-webcomponents-react/assets/148044451/bfa3db7d-d8c9-44be-9a9b-784d185c2391)

**Comment by Lukas742** [comment]
Hi @manoj23sap 

I'm going to close this Discussion, as it's lacking context (no code example, no explanation on how you're applying the styles, etc.) and is created in the wrong category. If you want to report a bug, please open an [issue](https://github.com/SAP/ui5-webcomponents-react/issues/new/choose), if you only want to ask a question, use the "Q&A" category.
